Sources/Clappr_iOS/Classes/Plugin/Core/Drawer/BottomDrawerPlugin.swift

```diff
@@ -0,0 +1,126 @@
+class BottomDrawerPlugin: DrawerPlugin {
+    private var initialCenterY: CGFloat = .zero
+
+    override var position: DrawerPlugin.Position {
+        return .bottom
+    }
+
+    override var size: CGSize {
+        return CGSize(width: coreViewFrame.width, height: coreViewFrame.height/2)
+    }
+
+    private var minYToShow: CGFloat {
+        return coreViewFrame.height * 0.75
+    }
+
+    private var hiddenHeight: CGFloat {
+        return coreViewFrame.height - placeholder
+    }
+
+    required init(context: UIObject) {
+        super.init(context: context)
+
+        addTapGesture()
+        addDragGesture()
+    }
+
+    override func render() {
+        super.render()
+
+        adjustSize()
+        moveDown(with: .zero)
+        adjustInitialPosition()
+    }
+
+    private func adjustSize() {
+        view.frame.size = size
+    }
+
+    private func adjustInitialPosition() {
+        initialCenterY = view.center.y
+    }
+
+    override func onDrawerShow() {
+        moveUp()
+    }
+
+    override func onDrawerHide() {
+        moveDown()
+    }
+
+    private func moveUp() {
+        view.setVerticalPoint(to: size.height, duration: ClapprAnimationDuration.mediaControlShow)
+    }
+
+    private func moveDown(with duration: TimeInterval = ClapprAnimationDuration.mediaControlHide) {
+        view.setVerticalPoint(to: hiddenHeight, duration: duration)
+    }
+
+    private func addTapGesture() {
+        let gesture = UITapGestureRecognizer(target: self, action: #selector(didTapView))
+        view.addGestureRecognizer(gesture)
+    }
+
+    private func addDragGesture() {
+        let gesture = UIPanGestureRecognizer(target: self, action: #selector(onDragView))
+        view.addGestureRecognizer(gesture)
+    }
+
+    @objc private func didTapView() {
+        showDrawerPlugin()
+        hideMediaControl()
+    }
+
+    private func showDrawerPlugin() {
+        core?.trigger(.showDrawerPlugin)
+    }
+
+    private func hideMediaControl() {
+        activeContainer?.trigger(.disableMediaControl)
```

Comment:
since MediaControl listens to `. didShowDrawerPlugin ` in order to hide himself:
```swift
listenTo(core, event: .didShowDrawerPlugin) { [weak self] _ in
    self?.hide()
}
```

I think we can test out if we really need to trigger the `disableMediaControl ` event

Reply:
Nice.